serving/resource_model_serving_provisioned_throughput.go

```diff
@@ -0,0 +1,115 @@
+			common.CustomizeSchemaPath(m, "config", "traffic_config").SetComputed()
+			common.CustomizeSchemaPath(m, "config", "served_entities", "name").SetComputed()
+			common.CustomizeSchemaPath(m, "config", "served_entities", "provisioned_model_units").SetRequired()
+			common.CustomizeSchemaPath(m, "config", "served_entities", "entity_name").SetRequired()
+			common.CustomizeSchemaPath(m, "config", "served_entities", "entity_version").SetRequired()
+
+			m["serving_endpoint_id"] = &schema.Schema{
+				Computed: true,
+				Type:     schema.TypeString,
+			}
+			return m
+		})
+
+	return common.Resource{
+		Create: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+			w, err := c.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			var e serving.CreatePtEndpointRequest
+			common.DataToStructPointer(d, s, &e)
+			wait, err := w.ServingEndpoints.CreateProvisionedThroughputEndpoint(ctx, e)
+			if err != nil {
+				return err
+			}
+			endpoint, err := wait.GetWithTimeout(d.Timeout(schema.TimeoutCreate) - deleteCallTimeout)
+			if err != nil {
+				log.Printf("[ERROR] Error waiting for serving endpoint to be created: %s", err.Error())
+				nestedErr := w.ServingEndpoints.DeleteByName(ctx, e.Name)
+				if nestedErr != nil {
+					log.Printf("[ERROR] Error cleaning up serving endpoint: %s", nestedErr.Error())
+				}
+				return err
+			}
+			d.SetId(endpoint.Name)
+			return nil
+		},
+		Read: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+			w, err := c.WorkspaceClient()
+			var sOrig serving.ServingEndpointDetailed
+			common.DataToStructPointer(d, s, &sOrig)
+			if err != nil {
+				return err
+			}
+			endpoint, err := w.ServingEndpoints.GetByName(ctx, d.Id())
+			if err != nil {
+				return err
+			}
+			err = common.StructToData(*endpoint, s, d)
+			if err != nil {
+				return err
+			}
+			d.Set("serving_endpoint_id", endpoint.Id)
+			return nil
+		},
+		Update: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+			w, err := c.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			var e serving.CreatePtEndpointRequest
+			common.DataToStructPointer(d, s, &e)
+			if d.HasChange("config") {
+				var updateRequest serving.UpdateProvisionedThroughputEndpointConfigRequest
+				updateRequest.Name = e.Name
+				updateRequest.Config = e.Config
+
+				waiter, err := w.ServingEndpoints.UpdateProvisionedThroughputEndpointConfig(ctx, updateRequest)
+				if err != nil {
+					return err
+				}
+				_, err = waiter.GetWithTimeout(d.Timeout(schema.TimeoutUpdate))
+				if err != nil {
+					return err
+				}
+			}
+			return nil
+		},
+		Delete: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+			w, err := c.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			return w.ServingEndpoints.DeleteByName(ctx, d.Id())
+		},
+		StateUpgraders: []schema.StateUpgrader{},
```

Comment:
`StateUpgraders` is not required, `SchemaVersion` as well - they both will be initialized to zero values anyway